invenio_sipstore/archivers/bagit_archiver.py

```diff
@@ -82,8 +203,14 @@ def autogenerate_tags(self, files_info):
         self.tags['Payload-Oxum'] = '{0}.{1}'.format(
             sum([f['size'] for f in files_info.values()]), len(files_info))
 
+    def get_fetch_file(self, files_info):
+        """Generate the contents of the fetch.txt file."""
+        content = ('{0} {1} {2}'.format(c['path'], c['size'], f)
```

Comment:
I believe the FILENAME (and maybe even URL) as well must be url-encoded to avoid issues with spaces:

> If FILENAME begins with a slash character, the destination MUST still be treated as relative to the bag base directory.  One or more linear whitespace characters (spaces or tabs) MUST separate these three values, and any such characters in the URL MUST be percent-encoded [RFC3986].